src/util/UserFlags.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const BitField = require('./BitField');
+
+/**
+ * Data structure that makes it easy to interact with a {@link User#flags} bitfield.
+ * @extends {BitField}
+ */
+class UserFlags extends BitField {}
+
+/**
+ * @name UserFlags
+ * @kind constructor
+ * @memberof UserFlags
+ * @param {BitFieldResolvable} [bits=0] Bit(s) to read from
+ */
+
+/**
+ * Numeric user flags. All available properties:
+ * * `DISCORD_EMPLOYEE`
+ * * `DISCORD_PARTNER`
+ * * `HYPESQUAD_EVENTS`
+ * * `BUGHUNTER_LEVEL1`
+ * * `HOUSE_BRAVERY`
+ * * `HOUSE_BRILLIANCE`
+ * * `HOUSE_BALANCE`
+ * * `EARLY_SUPPORTER`
+ * * `TEAM_USER`
+ * * `SYSTEM`
+ * * `BUGHUNTER_LEVEL2`
+ * * `VERIFIED_BOT`
+ * * `VERIFIED_DEVELOPER`
+ * @type {Object}
+ * @see {@link https://discordapp.com/developers/docs/resources/user#user-object-user-flags}
+ */
+UserFlags.FLAGS = {
+  DISCORD_EMPLOYEE: 1 << 0,
+  DISCORD_PARTNER: 1 << 1,
+  HYPESQUAD_EVENTS: 1 << 2,
+  BUGHUNTER_LEVEL1: 1 << 3,
```

Comment:
```suggestion
  BUG_HUNTER_LEVEL_1: 1 << 3,
```
Minor nitpick but could the bug hunter flags use the above letter casing?